matchms/importing/load_from_json.py

```diff
@@ -55,11 +59,12 @@ def as_spectrum(dct: dict) -> Union[dict, Spectrum, None]:
     """
     # Recognize Spectrum by peaks_json key
     if 'peaks_json' in dct:
-        return dict2spectrum(dct)
+        return dict2spectrum(dct, harmonize_defaults=harmonize_defaults)
     return None
 
 
-def dict2spectrum(spectrum_dict: dict) -> Union[Spectrum, None]:
+def dict2spectrum(spectrum_dict: dict,
+                  harmonize_defaults: bool) -> Union[Spectrum, None]:
```

Comment:
What is the difference between `Union[Spectrum, None]` and `Optional[Spectrum]`?